bert_e/tests/assets/workflow.yml

```diff
@@ -0,0 +1,11 @@
+name: basic tests
+
+on: push
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - run: exit 0
```

Comment:
Could we add multiple jobs on this workflow, where you start with one, and when the first one finish you start the others. 

(or you can also add another workflow for that if you prefer)